src/main/java/io/antmedia/datastore/db/DataStore.java

```diff
@@ -1661,9 +1661,12 @@ public static Broadcast conferenceToBroadcast(ConferenceRoom conferenceRoom) thr
 	 * @param size 	number of items to get
 	 * @param role the role of the subtracks for role based streaming especially in conferences. It can be null
 	 * @param status the status of the stream broadcasting, finished etc. It can be null
+	 * @param sortBy can get "name" or "date" or "status" values
+	 * @param orderBy can get "desc" or "asc"
+	 * @param search is used for searching in streamIds and names of the stream
 	 * @return
 	 */
-	public abstract List<Broadcast> getSubtracks(String mainTrackId, int offset, int size, String role, String status);
+	public abstract List<Broadcast> getSubtracks(String mainTrackId, int offset, int size, String role, String status, String sortBy, String orderBy, String search);
```

Comment:
I'm assuming that no plugin is using the older method because it's breaking the backward compatibility